zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -105,6 +114,45 @@ void beforeEach() throws IOException {
     index = SchemaTestUtil.mockIndex("qualified_name", "alias", "index_name", "mappings.json");
   }
 
+  @Test
+  void shouldPeriodicallyFlushBasedOnConfiguration() {
+    // given
+    final var duration = 2;
+    config.bulk.setDelay(duration);
+
+    final var exporter = new CamundaExporter();
+
+    final var context =
+        new ExporterTestContext()
+            .setConfiguration(new ExporterTestConfiguration<>("elastic", config));
+
+    exporter.configure(context);
+
+    // when
+    final var spiedController = spy(controller);
+    exporter.open(spiedController);
+
+    spiedController.runScheduledTasks(Duration.ofSeconds(duration));
```

Comment:
❓ I'm curious what do we need this call for here?

Reply:
We don't use a real `controller` which submits jobs to the `actor` instead its a test controller where we need to manually call for the schduled tasks to run